server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const path = require('path');
+const port = process.env.PORT || 8080;
+
+const app = express();
+
+app.use(express.static(__dirname));
+
+app.get('*', (req, res) => {
+  res.sendFile(path.resolve(__dirname, 'index.html'));
+});
+
+app.listen(port);
+
+console.log('Server Started');
```

Comment:
nit: I think it'd be nice to echo out the port as well (usually want to echo something like "Server started - open http://localhost:{{port}}"